app/services/frontend_logger.rb

```diff
@@ -22,8 +22,6 @@ def track_event(name, attributes)
       )
     elsif analytics_method.respond_to?(:call)
       analytics_method.call(**hash_from_kwargs(attributes, analytics_method))
-    else
-      analytics.track_event("Frontend: #{name}", attributes)
```

Comment:
what if we had the method return `true` when it recognizes and `false` when it doesn't? then we could have the controller return like a 400 bad request if it does not log an event

Reply:
mad a simple attempt at this in a20cd2a47f